package.json

```diff
@@ -322,6 +322,7 @@
 		"stacktrace-gps": "3.0.2",
 		"stylelint": "9.10.1",
 		"supertest": "3.4.2",
+		"svgstore-cli": "^1.3.1",
```

Comment:
You might want to remove `ˆ`.

Reply:
Nah, you're right, I'll do it myself :slightly_smiling_face: